modules/integration_aws-backup/conf/00-aws-backup-failed.yaml

```diff
@@ -0,0 +1,22 @@
+module: AWS Backup
+name: failed
+id: backup_failed
+
+transformation: ".min(over='1h')"
+aggregation: true
+filtering: "filter('namespace', 'AWS/Backup')"
+value_unit: "count"
+
+signals:
+  failed:
+    metric: NumberOfBackupJobsFailed
+    extrapolation: zero
+    rollup: sum
+  signal:
+    formula: failed
+rules:
+  critical:
+    threshold: 0
+    comparator: ">"
+    lasting_duration: 2h
+    lasting_at_least: 0.9
```

Comment:
why this tolerance ? I think it should be a deterministic test here. the metric is a count not a ratio

Reply:
Addressed: https://github.com/claranet/terraform-signalfx-detectors/pull/457/files#diff-0eec7b99d6548a77ff0d378c3db0de6cae5c0527a186aeab8ed197d0e80904a3R16